readme.md

```diff
@@ -42,6 +45,70 @@ print(LaunchAtLogin.isEnabled)
 //=> true
 ```
 
+#### As singleton:
```

Comment:
This is useless to show as it's the same as the static property, but more verbose. This makes me think the current design is not optimal. We have to do an instance because of ObservableObject, but I'm thinking we can make the ObservableObject a separate class instead:

```swift
LaunchAtLogin.isEnabled // static property

LaunchAtLogin.observable // returns the shared class instance for the ObservableObject

LaunchAtLogin.publisher // returns the shared class property for the publisher.
```